pkg/mysql/topology/topology.go

```diff
@@ -0,0 +1,77 @@
+package topology
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+
+	apiv1alpha1 "github.com/percona/percona-server-mysql-operator/api/v1alpha1"
+	"github.com/percona/percona-server-mysql-operator/pkg/mysql"
+	"github.com/percona/percona-server-mysql-operator/pkg/orchestrator"
+	"github.com/percona/percona-server-mysql-operator/pkg/replicator"
+)
+
+type Topology struct {
+	Primary  string
+	Replicas []string
+}
+
+func Get(ctx context.Context, cluster *apiv1alpha1.PerconaServerMySQL, operatorPass string) (Topology, error) {
+	var err error
+	top := Topology{}
```

Comment:
we're initializing this variable only to reassign below
```suggestion
	var top Topology
```

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/167/commits/266a97034ba0b28a591bc0c6412a54d0dbbeaea2